### ISSUE TITLE
Little bug in the first level of "Frosty Frolics"

### ISSUE THREAD

**Issue by CHrvita** [Rinnegatamante/sleepwalkers-vita#2]
*Testing the Dream 6 of the World: "Frosty Frolics" if you try to join the "vines" of the beginning of the level (the ones closest to the initial spawn of the character) the game crashes. Maybe it's something random since the level can be completed without problem.

**Comment by Rinnegatamante** [comment]
Can you share a save to reproduce the issue?

**Comment by CHrvita** [comment]
> Can you share a save to reproduce the issue?

Yeah. Excuse my clumsiness. Not knowing exactly the file you need, I have sent you the entire data.
 https://www.mediafire.com/file/kfdl0l9zf09oi3m/sleepwalkers.rar/file

**Comment by Rinnegatamante** [comment]
![immagine](https://user-images.githubusercontent.com/9152710/224704904-fc82817f-7314-4bb4-8206-7c3dc634dcdd.png)
Doesn't seem to happen here.

**Comment by CHrvita** [comment]
If you throw the character and then pull the creeper it crashes.

**Comment by CHrvita** [comment]
If you throw the character and then pull the vine it crashes.

**Comment by Rinnegatamante** [comment]
![immagine](https://user-images.githubusercontent.com/9152710/224767765-b8a4a52a-1867-4714-a6f2-9d8deaf88df3.png)
Still didn't crash here.

**Comment by CHrvita** [comment]
https://www.mediafire.com/file/dxfcl1kp2pxzrht/test.rar/file

It is a video of the detected problem.
I hope that helps. And your work is very good.

**Comment by Rinnegatamante** [comment]
This seems to be a bug in the original game. Can you check if it happens on a regular phone as well?

**Comment by CHrvita** [comment]
I can not do it. Even so, I repeat: removing that small bug, (which, as you say, could be due to a bug in version 1.2 of the original game), yesterday I finished testing it (complete) and the performance is excellent.
